### PR TITLE
Set up an upstream node

### PR DESCRIPTION
**Pull request by sjnam**
## Proposed changes

- This article explains how to set up an upstream node of a full mode endpoint node.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [x] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- https://github.com/klaytn/klaytn/pull/2013

### CODE REVIEW

**Comment by sjnam** [comment]
recheck

File: docs/misc/operation/upstream-en.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Set up the upstream node of the full endpoint node
```
Comment (scott-klaytn) [inline docs/misc/operation/upstream-en.md:1]: Suggestion for title "Link Full Node to Archive"

File: docs/misc/operation/upstream-en.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+In order to query the state of a random block to a node, the node must have the state trie of the block. Archive nodes store the state of all blocks in the database, but full nodes store the state of the latest 128 blocks and at a 128 block cycle. For example, a full node can query the balance for blocks multiple of 128.
+Since the archive node stores all states in the database, the data size is very large, operation costs are high, and status inquiry also takes more load than a full node.
+If you want to check the status of all blocks while operating a full node, set the upstream node of the full node as an archive node, and the full node will request the status that it does not have from the archive node, which is an upstream node, and send the results obtained as a response to the request in the first place.
+
+## How to set up
+Set `--upstream-en` in the `ADDITIONAL` environment variable of `kend.conf`, the configuration file of the operating full node. The example below shows how to set the archive node `https://archive-en.cypress.klaytn.net` as an upstream node.
```
Comment (scott-klaytn) [inline docs/misc/operation/upstream-en.md:8]: Suggestion: Insert "To configure a full node to retrieve data from an upstream archive node, " before "Set ...".

**Review by scott-klaytn — COMMENTED** [review]
Other than the comments I left on "upstream-en.md", you need to add the entry to sidebar.

1. Open sidebars.js in the root.
2. Add the "upstream-en" entry under commonSidebar as shown in the attachment
upstream-en
.

**Comment by sjnam** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by scott-klaytn — APPROVED** [review]

**Review by ayo-klaytn — APPROVED** [review]

**Review by yoomee1313 — APPROVED** [review]

**Pull request merged**